### DISCUSSION TITLE
Connect to mySQL for user

### DISCUSSION THREAD

**Discussion by nopal98** [prasathmani/tinyfilemanager#653] [Q&A] [answered]
Hello,
I have Database, also Table User. Is it possible to integrate my User Table (in mySQL) for Login??

**Comment by prasathmani** [comment, marked as answer]
File manager does not supported login using database.  but still you can customize.

**Reply by sadmansikder** [reply]
ok I did that connecting it to localhost mysql server.
Get the codes here: https://github.com/sadmansikder/paperlab
(I have customized it little for my uses. So, never mind)

Re: Start with creating the database name as 'studio_error' and import the `.sql` file from the code repo. and create your user account and go for it.
